services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/permissions/HankeKayttaja.kt

```diff
@@ -41,13 +40,12 @@ data class HankeKayttajaDto(
     @field:Schema(description = "Email address") val sahkoposti: String,
     @field:Schema(description = "First name") val etunimi: String,
     @field:Schema(description = "Last name") val sukunimi: String,
-    @Deprecated("Use etunimi and sukunimi instead.")
-    @field:Schema(description = "Full name", deprecated = true)
-    val nimi: String,
     @field:Schema(description = "Phone number") val puhelinnumero: String,
     @field:Schema(description = "Access level in Hanke") val kayttooikeustaso: Kayttooikeustaso?,
     @field:Schema(description = "User roles in Hanke") val roolit: List<ContactType>,
     @field:Schema(description = "Has user logged in to view Hanke") val tunnistautunut: Boolean,
+    @field:Schema(description = "When their latest invitation was sent")
+    val kutsuttu: OffsetDateTime?,
```

Comment:
I guess that `kutsuttu` is never `null` for any one else than the founder of the hanke? And the founder never needs a tooltip where this information would be shown. So, the UI does not need to bother with null values for users that should show the invitation date, right?

Reply:
The invitation is deleted when accepted, so `kutsuttu` will be null for all identified users. There should be a value for all unidentified users since it's a non-nullable field in the invitation and all users should have exactly one of either permissions or invitations.